pxtblocks/blocklyloader.ts

```diff
@@ -2852,7 +2852,7 @@ namespace pxt.blocks {
             }
 
             let button = goog.dom.createDom('button') as HTMLElement;
-            button.setAttribute('text', lf("Make a Variable"));
+            button.setAttribute('text', lf("Make a Variable") + "...");
```

Comment:
Check RTL rendering

Reply:
Ahhh yes. I knew this hack was a bad idea. Should I do RTL or should I just add it to the Lf?

Reply:
just add it to the string.